modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/OnRevisionAppliedCallback.java

```diff
@@ -29,7 +29,7 @@ public interface OnRevisionAppliedCallback {
     /**
      * Notifies of completion of processing of Meta Storage watches for a particular revision.
      *
-     * @param watchEvent Event with modified Meta Storage entries processed at least one Watch.
```

Comment:
According to the ticket description, this does not need to be corrected in this ticket.

Reply:
And what do you propose? To remove this change?

Reply:
yep

Reply:
Why?